t/500_ping_icmp.t

```diff
@@ -22,7 +22,7 @@ my $is_devel = $ENV{PERL_CORE} || -d ".git" ? 1 : 0;
 $ENV{TEST_PING_HOST} = "127.0.0.1" if $ENV{NO_NETWORK_TESTING};
 # Note this rawsocket test code is considered anti-social in p5p and was removed in
 # their variant.
-# See http://nntp.perl.org/group/perl.perl5.porters/240707
+# See See https://www.nntp.perl.org/group/perl.perl5.porters/2016/11/msg240707.html
```

Comment:
Not really

Reply:
well, the link doesn't take you to the email it references. 
At any rate, this is pretty minor, I only came across it because I am currently in testing for a pull request that will use IPPROTO_ICMP socket type if they are available. This will allow unprivileged users to send an icmp ping without the need for SOCK_RAW. 
Cf: https://lwn.net/Articles/420800/

It's been in the kernel for quite some time, at least since 3.11 (which is the oldest kernel I have running).
There is a kernel parameter, net.ipv4.ping_group_range, which describes the gid of users allowed to use it. It's been set to 1,0 by default but distributions are setting it to its max value now (Fedora started in 31 I believe) and letting any user use it. 
So I figured the time was probably right to let perl try and use it if its available.